labels.rb

```diff
@@ -11,32 +12,32 @@
   opts.on('-n', '--namespace NAME', 'GitHub namespace. Required.') { |v| options[:namespace] = v }
   opts.on('-r', '--repo-regex REGEX', 'Repository regex') { |v| options[:repo_regex] = v }
   opts.on('-t', '--oauth-token TOKEN', 'OAuth token. Required.') { |v| options[:oauth] = v }
-  opts.on('-f', '--fix-labels', 'Add the missing labels to repo') { options[:fix_labels] = true}
+  opts.on('-f', '--fix-labels', 'Add the missing labels to repo') { options[:fix_labels] = true }
   opts.on('-d', '--delete-labels', 'Delete unwanted labels from repo') { options[:delete_labels] = true }
 
   # default filters
-  opts.on('--puppetlabs', 'Select Puppet Labs\' modules') {
+  opts.on('--puppetlabs', 'Select Puppet Labs\' modules') do
     options[:namespace] = 'puppetlabs'
     options[:repo_regex] = '^puppetlabs-'
-  }
+  end
 
-  opts.on('--puppetlabs-supported', 'Select only Puppet Labs\' supported modules') {
+  opts.on('--puppetlabs-supported', 'Select only Puppet Labs\' supported modules') do
     options[:namespace] = 'puppetlabs'
     options[:repo_regex] = OctokitUtils::SUPPORTED_MODULES_REGEX
-  }
+  end
 
-  opts.on('--voxpupuli', 'Select voxpupuli modules') {
+  opts.on('--voxpupuli', 'Select voxpupuli modules') do
     options[:namespace] = 'voxpupuli'
     options[:repo_regex] = '^puppet-'
-  }
+  end
 end
```

Comment:
Default filters can be removed as the initial data should be received from modules.json